dirty_cat/super_vectorizer.py

```diff
@@ -250,20 +276,35 @@ def _clone_transformers(self):
             self.low_card_cat_transformer_ = clone(self.low_card_cat_transformer)
         elif self.low_card_cat_transformer is None:
             self.low_card_cat_transformer_ = OneHotEncoder()
+        elif self.low_card_cat_transformer == "remainder":
+            self.low_card_cat_transformer_ = self.remainder
         else:
             self.low_card_cat_transformer_ = self.low_card_cat_transformer
 
         if isinstance(self.high_card_cat_transformer, sklearn.base.TransformerMixin):
             self.high_card_cat_transformer_ = clone(self.high_card_cat_transformer)
         elif self.high_card_cat_transformer is None:
             self.high_card_cat_transformer_ = GapEncoder(n_components=30)
+        elif self.high_card_cat_transformer == "remainder":
+            self.high_card_cat_transformer_ = self.remainder
         else:
             self.high_card_cat_transformer_ = self.high_card_cat_transformer
 
+        if isinstance(self.numerical_transformer, sklearn.base.TransformerMixin):
+            self.numerical_transformer_ = clone(self.numerical_transformer)
+        elif self.numerical_transformer is None:
+            self.numerical_transformer_ = "passthrough"
+        elif self.numerical_transformer == "remainder":
+            self.numerical_transformer_ = self.remainder
+        else:
+            self.numerical_transformer_ = self.high_card_cat_transformer
```

Comment:
I'm confused here. If I read this correctly, we are applying a transformer designed for categorical data to numerical features? This seems incorrect.

Reply:
It's an error indeed, see message above :)